pkg/aws_config_client/completer.go

```diff
@@ -252,10 +292,17 @@ func (c *completer) Complete(out *ini.File) error {
 	if err != nil {
 		return err
 	}
+	if len(profiles) == 0 {
+		logrus.Info("No AWS profiles configured")
+		return nil
+	}
 
-	err = c.writeAWSProfiles(out, region, profiles)
+	newAWSProfiles, err := c.assembleAWSConfig(region, profiles)
 	if err != nil {
 		return err
 	}
-	return nil
+
+	mergedConfig, err := c.mergeConfigs(newAWSProfiles, base)
+	writeCode, err := mergedConfig.WriteTo(w)
```

Comment:
@edulop91 This line panics when mergedConfig is nil. That's why I've been returning ini.Empty() if I get an error

Reply:
ah, it should not be a problem when you err check 305 though!

Reply:
This is super nitpicky, but if someone types "no" to "does this config file look right?", should I return an error? I discovered this issue when I answered "no".